js/ui/date_box/ui.date_box.base.js

```diff
@@ -693,6 +693,12 @@ const DateBox = DropDownEditor.inherit({
             case 'todayButtonText':
                 this._setPopupOption('toolbarItems', this._getPopupToolbarItems());
                 break;
+            case 'dropDownOptions':
+                this.callBase.apply(this, arguments);
+                if(this._strategy._timeView) {
```

Comment:
We have this [[error](https://github.com/DevExpress/DevExtreme/actions/runs/4562599836/jobs/8050824025)] without it condition. Any ways to solve this otherwise?

Reply:
Absolutely agree @ksercs. This part of the code was not completed. This has been fixed in: [8a57e6599443afaa0f43c87293b16ebc52de9546].